templates/expo-template-bare-minimum/android/gradle.properties

```
@@ -41,6 +41,11 @@ newArchEnabled=true
 # If set to false, you will be using JSC instead.
 hermesEnabled=true
 
+# Use this property to enable edge-to-edge display support.
+# This allows your app to draw behind system bars for an immersive UI.
+# Note: Only works with ReactActivity and should not be used with custom Activity.
+edgeToEdgeEnabled=false
```

I believe we should default this to true, given that it's already true in your `app.json` templates

```suggestion
edgeToEdgeEnabled=true
```